src/targets/github.ts

```diff
@@ -336,7 +328,7 @@ export class GitHubTarget extends BaseTarget {
    *
    * @param release Release object
    */
-  public async publishRelease(release: GitHubRelease) {
+  public async publishRelease(release: GitHubRelease, isLatest = true) {
```

Comment:
Same concern as above re boolean args and also naming. Would prefer something like `setAsLatest` or `makeLatest`